src/SignUp.tsx

```diff
@@ -68,20 +57,41 @@ const SignUp = () => {
 
 	const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
 		e.preventDefault();
+		const target = e.target as FormFieldsType;
+
+		const firstName = target.firstName.value;
+		const lastName = target.lastName.value;
+		const email = target.email.value;
+		const password = target.password.value;
+		const confirmPassword = target.confirmPassword.value;
 		if (isValid(e)) {
-			axios.post("http://localhost:8080/customers", values).catch(errors => {
-				if (errors.toJSON().message === "Network Error") {
-					setError("Error... Something gone wrong");
-				}
-			});
-			navigate("/login");
+			axios
+				.post("http://localhost:8080/customers", {
+					firstName,
+					lastName,
+					email,
+					password,
+					confirmPassword,
+				})
+				.then(() => {
+					setSuccessful("Your sign-up as been succesfull. Please log-in");
+					setTimeout(() => {
+						navigate("/login");
+					}, 2500);
```

Comment:
we don't need to wait 2.5 secs to route the user to the /login

Reply:
You are right, i will remove it